compiler/luci/pass/src/RemoveRedundantTranspose.test.cpp

```diff
@@ -105,6 +114,112 @@ void create_redundunt_transpose(loco::Graph *g, const std::vector<int32_t> &perm
   auto graph_output = g->outputs()->create();
   output->index(graph_output->index());
 }
+/**
+ *  BEFORE
+ *            |
+ *      [CircleNode]       [CircleConst]
+ *               \           /
+ *[CircleConst] [CircleTranspose] [CircleConst]
```

Comment:
Can you add some spaces between `*` and `[CircleConst]` ?

Reply:
Okay I will fix it.